tests/state_test.go

```diff
@@ -706,6 +706,49 @@ func TestAllETH(t *testing.T) {
 	skipTests["RevertPrefoundEmptyOOG.json"] = "State trie clearing unimplemented"
 	skipTests["FailedCreateRevertsDeletion.json"] = "State trie clearing unimplemented"
 
+	//Constantinople gas calculation skipped tests
+	skipTests["extCodeHashAccountWithoutCode.json/Constantinople/0"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashDynamicArgument.json/Constantinople/0"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashDynamicArgument.json/Constantinople/1"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashDynamicArgument.json/Constantinople/2"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashDynamicArgument.json/Constantinople/3"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashDynamicArgument.json/Constantinople/4"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashNonExistingAccount.json/Constantinople/0"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/0"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/1"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/2"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/3"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/4"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/5"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/6"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/7"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashPrecompiles.json/Constantinople/8"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashSubcallOOG.json/Constantinople/0"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashSubcallOOG.json/Constantinople/1"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashSubcallOOG.json/Constantinople/2"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashSubcallOOG.json/Constantinople/3"] = "Unsupported Gas Calculations"
+	skipTests["extCodeHashSubcallOOG.json/Constantinople/4"] = "Unsupported Gas Calculations"
```

Comment:
Wasn't there something about skipping constantinople altogether? Would save skipping all individual as there won't be any actual functional difference in what is tested between consta..fix and consta...